schema/instance.go

```diff
@@ -21,9 +23,8 @@ const (
 )
 
 type Instance struct {
```

Comment:
I know it will create a stutter, but still, maybe it would be better if we named this struct `Schema`. Wdyt?

Reply:
@lovromazgon probably makes sense. I personally liked how it read when using this as a exported struct (`schema.Instance`), but I guess it makes sense to at least add `Schema` since it is a representation of a `Schema`. I was also thinking `SchemaInstance`. Too much?

Reply:
@lovromazgon actually, it seems like this follows a pattern we've stablished in previous objects (https://github.com/ConduitIO/conduit/blob/00dbcba7d5b3728d6ec8204ca66756e816699b30/pkg/connector/instance.go#L49 for example)

Reply:
Yeah, I'm not convinced by my comment either 😄 Maybe `Instance` is better. Ok let's leave it 👍

Reply:
Ok, to follow up with a slack conversation. The references I made live in Conduit, so if we are embracing consistency, I'm fine with leaving both type and proto message with the same name and follow this type naming on conduit-commons moving forward. If we realize those that live in conduit need to adapt (or be refactored) then we'll see opportunities to adopt this "sutter" naming.